app/controllers/group_assignments_controller.rb

```diff
@@ -21,4 +21,35 @@ def create
       end
     end
   end
+
+  def destroy
+    @meeting = Meeting.find(params["meeting_id"])
+    @individual = Individual.find(params["individual_id"])
+
+    if params["group_assignment"]["group"].empty?
```

Comment:
this code makes an assumption that `params["group_assignment"]` is not null. Consider:

```
if params["group_assignment"].present? && params["group_assignment"]["group"]
```